apps/full-site-editing/full-site-editing-plugin/jetpack-timeline/blocks/src/timeline-item.js

```diff
@@ -1,24 +1,63 @@
 /**
  * External dependencies
  */
-
-import { InspectorControls, InnerBlocks, PanelColorSettings } from '@wordpress/block-editor';
+import {
+	InspectorControls,
+	InnerBlocks,
+	PanelColorSettings,
+	BlockControls,
+} from '@wordpress/block-editor';
 import { registerBlockType } from '@wordpress/blocks';
+import { ToolbarButton, ToolbarGroup } from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
 import { __ } from '@wordpress/i18n';
+import { positionLeft, positionRight } from '@wordpress/icons';
+import classnames from 'classnames';
 
 /**
  * Internal dependencies
  */
 import { TimelineIcon } from './icon';
 
+function Controls( { alignment, clientId, toggleAlignment } ) {
+	const parentIsAlternating = useSelect( ( select ) => {
+		const parentIds = select( 'core/block-editor' ).getBlockParents( clientId );
+		const parent = select( 'core/block-editor' ).getBlock( parentIds[ 0 ] );
+		return parent?.attributes?.isAlternating;
+	} );
+
+	if ( parentIsAlternating === false ) {
+		return null;
+	}
+
+	return (
+		<BlockControls>
+			<ToolbarGroup>
+				<ToolbarButton
+					onClick={ () => toggleAlignment( 'left' ) }
+					isActive={ alignment === 'left' }
+					icon={ positionLeft }
+					title={ __( 'Left', 'full-site-editing' ) }
+				/>
+				<ToolbarButton
+					onClick={ () => toggleAlignment( 'right' ) }
+					isActive={ alignment === 'right' }
+					icon={ positionRight }
+					title={ __( 'Right', 'full-site-editing' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **162** times:
`translate( 'Right' )` *ES Score: 10*